<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFilesSearchWindow</class>
 <widget class="QMainWindow" name="CFilesSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
      <item row="3" column="1">
       <widget class="CHistoryComboBox" name="searchRoot">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Search in</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CHistoryComboBox" name="nameToFind">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnSearch">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>With contents</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seacrh for</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CHistoryComboBox" name="fileContentsToFind">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="progressLabel">
      <property name="text">
       <string>Searching...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="resultsList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CHistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/chistorycombobox.cpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
